### ISSUE TITLE
Suggested username(s) not accessible through screen reader

### ISSUE THREAD

**Issue by fwiep** [keepassxreboot/keepassxc-browser#2084]
## Expected Behavior
When an unlocked database is connected, a login form is opened and the username-field has focus, the suggested username(s) should be spoken aloud by the screen reader, when selected.

## Current Behavior
The suggested username(s) are not spoken by the screen reader at all.

## Steps to Reproduce (for bugs)
1. Unlock an existing database with connected browser plugin
2. Visit a site having one or more logins saved in the database
3. Focus, for example, the username-field.
4. Push <kbd>Arrow-down</kbd> to show the suggested username(s)

## Debug info
There are no console messages or errors.

KeePassXC - Version 2.7.6
Revision: dd21def
Qt 5.15.11
Debugging mode is disabled.

Operating system: Fedora Linux 39 (Workstation Edition)
CPU architecture: x86_64
Kernel: linux 6.6.9-200.fc39.x86_64

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare
- YubiKey
- Secret Service Integration

Cryptographic libraries:
- Botan 2.19.3

KeePassXC-Browser Version: 1.8.11 
Browser: Firefox 121.0
Screen reader: Orca `main` (rev. `097f83e3bfdf09ef58814ebe3041c6543edc7ca6`)

**Comment by varjolintu** [comment]
Not sure if the data is accesible because it's behind a closed Shadow DOM. Must investigate this further.
